.coveragerc

```diff
@@ -8,11 +8,13 @@ omit =
     # TODO(rhcarvalho): this is used to ignore test files from coverage report.
     # We can make this less generic when we stick with a single test pattern in
     # the repo.
-    test_*.py
-    *_tests.py
+    conftest.py
+    */conftest.py
```

Comment:
Indeed the current pattern is not omitting what it should, thanks for noticing it!

I see that `coverage` is using [`fnmatch`](https://docs.python.org/2/library/fnmatch.html) under the hoods, these should cover all cases, tested:

```
    */test_*.py
    */*_tests.py
    */conftest.py
```

These patterns get translated to regexps, e.g.:

```python
In [13]: fnmatch.translate('*/*_tests.py')
Out[13]: '.*\\/.*_tests\\.py\\Z(?ms)'
```

And coverage is testing against the full path for paths that start with a wildcard, or relative to the current dir for others.
https://coverage.readthedocs.io/en/latest/source.html#execution

Both `*/conftest.py` and `conftest.py` match `conftest.py` in the repo root.